File: problems/p/pet/pet.py
#! /usr/bin/python3
"""
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
"""

def main():
    winner = [0, 0]
    for i in range(5):
        grade = sum([int(val) for val in input().split()])
        if winner[1] < grade:
            winner[0] = i + 1
            winner[1] = grade
    print(f"{winner[0]} {winner[1]}")


if __name__ == '__main__':
    main()

File: problems/p/pet/pet.c
/*
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
*/

#include <stdio.h>

#define CONTESTANTS_SIZE 5


int main(void) {
    int grade_1, grade_2, grade_3, grade_4, winner_id, winner_points, points;
    winner_id = 0;
    winner_points = 0;
    for (int i=0; i<CONTESTANTS_SIZE; i++) {
        if (scanf("%d %d %d %d\n", &grade_1, &grade_2, &grade_3, &grade_4) != 4) {
            printf("Failed to read grades!");
        }

        points = grade_1 + grade_2 + grade_3 + grade_4;

        if (winner_points < points) {
            winner_id = i + 1;
            winner_points = points;
        }
    }

    printf("%d %d", winner_id, winner_points);

    return 0;
}